a10_octavia/controller/worker/tasks/vthunder_tasks.py

```diff
@@ -164,9 +164,7 @@ def execute(self, vthunder, loadbalancer, added_ports, ifnum_master=None, ifnum_
 
         if not lb_exists_flag:
             added_ports[amphora_id] = []
-            for nic in nics:
-                if nic.network_id == loadbalancer.vip.network_id:
-                    added_ports[amphora_id].append(nic)
+            added_ports[amphora_id].append(nics[1])
```

Comment:
`added_ports[amphora_id].append(nics[1])` will also not be needed here as we are already injecting None to the `added_ports` in `EnableInterface` of 1st loadbalancer.

Reply:
tried. All interface will not be enabled if we remove this.

```
interface ethernet 1/1
!
interface ethernet 1/2
!
interface ethernet 1/3
!
interface ethernet 2/1
!
interface ethernet 2/2
!
interface ethernet 2/3
!
vrrp-a vrid 0
  floating-ip 192.168.91.125
!
vrrp-a vrid 1
  device-context 1
    blade-parameters
      priority 150
!
health monitor octavia_health_monitor
  retry 5
vThunder-vMaster[1/1](NOLICENSE)#show in
vThunder-vMaster[1/1](NOLICENSE)#show interfaces br
vThunder-vMaster[1/1](NOLICENSE)#show interfaces brief
Port                Link  Dupl  Speed  Trunk Vlan Encap     MAC             IP Address          IPs Flags  Name
---------------------------------------------------------------------------------------------------------------
mgmt                Up    auto  auto   N/A   N/A  N/A       fa16.3e81.fca2  192.168.0.194/24      1
1                   Disb  None  None   none  1    N/A       fa16.3e15.81f8  0.0.0.0/0             0
2                   Disb  None  None   none  1    N/A       fa16.3ea2.df6b  0.0.0.0/0             0
3                   Disb  None  None   none  1    N/A       fa16.3e2c.f073  0.0.0.0/0             0


```